Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/PokePointer.cs

```diff
@@ -13,7 +13,25 @@ public class PokePointer : BaseControllerPointer, IMixedRealityNearPointer
 
         [SerializeField]
         protected GameObject visuals;
-        
+
+        /// <summary>
+        /// Maximum distance a which a touchable surface can be interacted with.
+        /// </summary>
+        [SerializeField]
+        protected float touchableDistance = 0.2f;
+        public float TouchableDistance => touchableDistance;
+
+        /// <inheritdoc />
+        private LayerMask[] pokeLayerMasks = null;
+        public LayerMask[] PokeLayerMasks => pokeLayerMasks;
```

Comment:
If this is not serialized, can we do this instead?
public PokeLayerMasks LayerMask[] { get; private set; }?

Reply:
Yes, good point.